src/hooks/useTxHistory.ts

```diff
@@ -47,7 +46,7 @@ const useTxHistory = (
       page: dataPage
         ? {
             ...dataPage,
-            results: dataPage.results.filter(filterNoNonceTransfers).filter(filterEmptyLabels),
```

Comment:
We should keep `filterEmptyLabels`

Reply:
See the edit, it's actually not needed now since all the txs are displayed again.

Reply:
Reverted the revert :D

Reply:
We can at this point just return data page without the restructuring.